modules/apps/document-library/document-library-web/src/main/resources/META-INF/resources/document_library/search_resources.jsp

```diff
@@ -68,7 +66,6 @@ entriesChecker.setRememberCheckBoxStateURLRegex("^(?!.*" + liferayPortletRespons
 
 	<div class="document-container" id="<portlet:namespace />entriesContainer">
 		<liferay-ui:search-container
-			emptyResultsMessage='<%= LanguageUtil.format(request, "no-documents-were-found-that-matched-the-keywords-x", HtmlUtil.escape(keywords), false) %>'
```

Comment:
Is this correct? Do we really want to remove the empty message?

Reply:
The empty message is removed here in the JSP, but is added during the creation of the `searchContainer` in JAVA
Reason: more logic is needed to decide which message to show 😄

Reply:
Thanks for the clarification, I didn't review the JAVA code 🙈